src/glomap/estimators/global_positioning.h

```diff
@@ -58,6 +58,10 @@ struct GlobalPositionerOptions {
   // Scaling factor for the loss function
   double loss_function_scale = 0.1;
 
+  // Whether to use custom parameter block ordering for Schur-based solvers.
+  // Disable for deterministic behavior when using a fixed random seed.
```

Comment:
It's unclear to me why a fixed parameter block ordering will make the tests non-deterministic? Sounds like a bug on the ceres side? Wouldn't we expect it to be the vice versa?